src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,44 @@
+package mate.academy.service
+
+import mate.academy.model.Circle
+import mate.academy.model.Color
+import mate.academy.model.IsoscelesTrapezoid
+import mate.academy.model.Rectangle
+import mate.academy.model.RightTriangle
+import mate.academy.model.Square
+import kotlin.random.Random
+
+const val MIN_RANDOM_SIZE = 1.0
+const val MAX_RANDOM_SIZE = 10.0
+
+class FigureSupplier {
+
+    private val colorSupplier = ColorSupplier()
+
+    private fun getRandomFigureType(): FigureType {
```

Comment:
Please check if this method could be replaced with private variable that contains enum's values and random() call in getRandomFigure() function when block